public/calendar_reboot.js

```diff
@@ -136,20 +130,20 @@ CheckIt.prototype.displayActiveCalendar = function() {
     // Display the active calendar if there is one.
       // Get the current active calendar from storage and display it.
     // If there is none, show build calendar menu.
-   this.store.getActive()
+   this.firebaseStore.getActive()
        .then(function (activeCalendarId) {
            
-           this.store.loadById(activeCalendarId)
+           this.firebaseStore.loadById(activeCalendarId)
                .then(function (activeCalendarState) {
                    if (activeCalendarState !==  null) {
                        var state = activeCalendarState;
                        var calendar = new Calendar(state, this);
                        this.displayCalendar(calendar);
                    }
                }.bind(this))
-               .catch(function() {
-                   console.error("Calendar does not exist");
-                   this.store.removeActive();
+               .catch(function(err) {
+                   console.error("Calendar does not exist " + err);
```

Comment:
Are you certain this means that the calendar doesn't exist? What if firebase itself just had a temporary problem?